tests/fullstack-test/expr/reverse.test

```diff
@@ -20,10 +20,16 @@ mysql> insert into test.t values('one week’s time test');
 mysql> insert into test.t values('abc测试def');
 mysql> insert into test.t values('abcテストabc');
 mysql> insert into test.t values('ѐёђѓєѕіїјљњћќѝўџ');
-mysql> insert into test.t values('+ѐ-ё*ђ/ѓ!є@ѕ#і@ї%ј……љ&њ（ћ）ќ￥ѝ#ў@џ！^');
+mysql> insert into test.t values('+ѐ-ё*ђ/ѓ!є@ѕ#і$@ї%ј……љ&њ（ћ）ќ￥ѝ#ў@џ！^');
 mysql> insert into test.t values('αβγδεζηθικλμνξοπρστυφχψωσ');
 mysql> insert into test.t values('▲α▼βγ➨δε☎ζη✂θι€κλ♫μν✓ξο✚πρ℉στ♥υφ♖χψ♘ω★σ✕');
 mysql> insert into test.t values('թփձջրչճժծքոեռտըւիօպասդֆգհյկլխզղցվբնմշ');
+mysql> insert into test.t values('\$$\$$\$$');
+mysql> insert into test.t values('\$$\$$\$$\$');
+mysql> insert into test.t values('\`\$$\$$\$');
+mysql> insert into test.t values('\`$$`\$$\$$\$');
+mysql> insert into test.t values('\\$```');
+mysql> insert into test.t values('\\\```');
```

Comment:
explanation for the added tests:

1. `\$$\$$\$$`: note that `\` is also a escape character for mysql, therefore mysql will only insert `$$$$$$` into the table
2. similar to the first case, mysql will only insert `$$$$$$$` into the table
3. similar to the first case, mysql will only insert `$$$$$ into the table
4. similar to the first case, mysql will only insert \`$$ `$$$$$ into the table
5. mysql will insert \\$``` into the table, however since we don't use `--raw` option for mysql client, the output backslash will be escaped to be double backslash
6. mysql will insert \\``` into the table, however since we don't use `--raw` option for mysql client, the output backslash will be escaped to be double backslash

ref: https://dev.mysql.com/doc/refman/8.0/en/mysql-command-options.html#option_mysql_raw